src/sfctl/telemetry.py

```diff
@@ -85,6 +87,56 @@ def check_and_send_telemetry(args_list, invocation_ret_val, exception=None):
             logger.info(
                 str.format('Not sending telemetry because python process due to error: {0}', ex))
 
+    # Prompt the user if there is no value set in the config file for whether or not telemetry
+    # should be sent. If use telemetry is set to False, then do nothing.
+    elif use_telemetry is None:
+
+        _prompt_for_telemetry(args_list, invocation_ret_val, exception)
+
+
+def _prompt_for_telemetry(args_list, invocation_ret_val, exception):
+    """
+    Ask users for a yes or no answer on collecting telemetry.
+
+    If the user says yes, set the value in config, then re-run the check_and_send_telemetry check.
+    If the user says no, set the value in config and return.
+
+    Inputs are the ones passed into check_and_send_telemetry. This just passes the information
+    along.
+
+    :return: None
+    """
+
+    user_response = None
+
+    while user_response not in ['y', 'yes', 'n', 'no']:
```

Comment:
Does knack have any prompting features we should be using for this?

Reply:
I'm not sure. Will look it up before merging this change in - this change will not be part of 7.1.0 

Thanks!